docs/content/latest/develop/query-1-performance/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Query Performance
+headerTitle: Query Performance
+linkTitle: Query Performance
+description: Improve query performance via optimization
+headcontent: Improve query performance via optimization.
+image: /images/section_icons/index/develop.png
+aliases:
+  - /develop/query-1-performance/
+section: Develop
+menu:
+  latest:
+    identifier: query-1-performance
+    parent: develop    
+    weight: 600
+---
+
+<div class="row">
+
+
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="learn/">
```

Comment:
This is a broken link... should be `href="explain-analyze/"`

Reply:
This is not in the source file. It's generated by Hugo, I suppose. I'm not sure how to fix it.

Reply:
The path for this page is under `query-1-performance`. I think it would be more consistent with the other documentation to place it under `query-performance`.

I'm not familiar with how Hugo generates these paths/links, either, so I am not certain what to suggest for you to do here.

Reply:
We can't do that. The dir names must be globally unique, and there's already a dir somewhere named query-performance. Do you have any more comments regarding the content of the doc?

Reply:
Fixed the broken link. Will upload the new version in a minute

Reply:
Looks good, thanks!